packages/trader/src/AppV2/Containers/Trade/trade.tsx

```diff
@@ -1,13 +1,191 @@
 import React from 'react';
-import { Text } from '@deriv-com/quill-ui';
+import { ActionSheet } from '@deriv-com/quill-ui';
+import { observer } from '@deriv/stores';
 import BottomNav from 'AppV2/Components/BottomNav';
+import { DraggableList } from 'AppV2/Components/DraggableList';
+import { TradeTypeList } from 'AppV2/Components/TradeTypeList';
+import { useTraderStore } from 'Stores/useTraderStores';
+import { Localize } from '@deriv/translations';
+
+const Trade = observer(() => {
+    const trade_store = useTraderStore();
+    const { contract_types_list, prepareTradeStore } = trade_store;
+
+    type Category = {
```

Comment:
Are we always create types before (outside) the component? Or it's optional and we can create inside?

Reply:
I don't think  it matters much